### PR TITLE
#6638 fixed table display type conversion

### PR DESCRIPTION
**Pull request by lmitusinski**

### CODE REVIEW

File: beakerx/beakerx/tabledisplay/tabledisplay.py [hunk 1]
```diff
@@ -121,7 +121,9 @@ def convert_from_pandas(self, args, types_map):
 
     @staticmethod
     def convert_value(value, value_type):
-        if value_type == "time":
+        if value == "":
```
Comment (jmsdnns) [inline beakerx/beakerx/tabledisplay/tabledisplay.py:124]: I'm curious if a falsy test would be helpful here. The current code checks specifically for value being an empty string, but if the check was `if value:` we would catch None, an empty list, and other falsy values too.

Not sure if that's desirable, but certainly worth mentioning.

Comment (scottdraves) [reply beakerx/beakerx/tabledisplay/tabledisplay.py:124]: empty list should appear as [].
none sounds right though, what does it do now?

Comment (jmsdnns) [reply beakerx/beakerx/tabledisplay/tabledisplay.py:124]: In the context of this function, it seems we return "string" for anything that doesn't match the specific checks. If we want to return "string" even when value is `None`, then the current code is all good. If we want to do something else when the value is `None`, we could do this at the at the end of the function.

Again, I'm not sure what the ideal behavior is, but I see a potentially unhandled case.

```
if value:
    return "string"
raise Exception("Unknown value type")
```

Comment (scottdraves) [reply beakerx/beakerx/tabledisplay/tabledisplay.py:124]: see also https://github.com/twosigma/beakerx/pull/6657 https://github.com/twosigma/beakerx/issues/6656

**Pull request merged**